test/e2e/framework/volume/fixtures.go

```diff
@@ -563,20 +563,20 @@ func testVolumeContent(client clientset.Interface, pod *v1.Pod, fsGroup *int64,
 // and check that the pod sees expected data, e.g. from the server pod.
 // Multiple Tests can be specified to mount multiple volumes to a single
 // pod.
-func TestVolumeClient(client clientset.Interface, config TestConfig, fsGroup *int64, fsType string, tests []Test) {
+func TestVolumeClient(f *framework.Framework, client clientset.Interface, config TestConfig, fsGroup *int64, fsType string, tests []Test) {
```

Comment:
/lgtm cancel

I think this is another case of a redundant `client` parameter. Remove the parameter and use `f.ClientSet` instead in the call to `runVolumeTesterPod`.

Reply:
made that change and pushed.
Should we also delete cs arg from test/e2e/storage/flexvolume.go:testFlexVolume, which now becomes unused. By same logic, passing cs is redundant already in current code if we assume it is always set to f.ClientSet, and framework arg is there now already

Reply:
> Should we also delete cs arg from test/e2e/storage/flexvolume.go:testFlexVolume, which now becomes unused.

Yes.

> By same logic, passing cs is redundant already in current code if we assume it is always set to f.ClientSet, and framework arg is there now already

Correct.